src/wp-content/themes/twentytwenty/assets/css/editor-style-block-rtl.css

```
@@ -1064,6 +1064,10 @@ hr.wp-block-separator.is-style-dots::before {
 	margin-top: 15px;
 }
 
+.editor-styles-wrapper .wp-block-latest-posts__post-full-content p {
```

```suggestion
.wp-block-latest-posts__post-full-content p {
```

These do not need the `.editor-styles-wrapper` class in either editor (post or widgets).